pkg/lwip/Makefile

```diff
@@ -1,11 +1,44 @@
 PKG_NAME=lwip
 PKG_URL=git://git.savannah.nongnu.org/lwip.git
-PKG_VERSION=STABLE-2_0_3_RELEASE
+# lwIP v2.1.0
+PKG_VERSION=e6a8415df332ee34d7af02255b2aa1e8ee74348f
 PKG_LICENSE=BSD-3-Clause
 
-.PHONY: all
+LWIP_SUBMODULES      := lwip_api lwip_core lwip_ipv4 lwip_ipv6 \
```

Comment:
For the variables you do not need the `:=` as there is no need for immediate evaluation, using `=` here is enough.
It does not change the behavior in this case but I am trying not to add new unnecessary immediate evaluation cases.